fiftyone/core/fields.py

```diff
@@ -23,7 +23,13 @@
 import fiftyone.core.utils as fou
 
 
-def validate_constraints(ftype=None, embedded_doc_type=None, read_only=None):
+def validate_constraints(
```

Comment:
might need to move this out of this PR or wait for https://github.com/voxel51/fiftyone/pull/4597 to merge